express/io_utils.py

```diff
@@ -0,0 +1,16 @@
+"""This module defines commpon I/O utilities."""
+import yaml
+import typing as t
+
+
+def load_yaml(yaml_path: str) -> t.Dict:
+    """
+    Loads a YAML file and returns a dictionary.
+    Args:
+        yaml_path (str): the path to the yaml file
+    """
+    try:
+        with open(yaml_path) as f:
+            return yaml.safe_load(f)
+    except Exception as e:
+        raise ValueError(f"Failed to load YAML file: {e}")
```

Comment:
Why do we throw a `ValueError` here instead of just raising the original yaml error??

Reply:
My bad, should be a general YAML error indeed

Reply:
Just re-raising the error doesn't provide a lot of value, right? Then we can just remove the `try ... except`, in which case the whole function doesn't really provide a lot of value.